examples/dev-mashup/index.js

```diff
@@ -0,0 +1,36 @@
+import { embed } from '@nebula.js/stardust';
+import bar from '@nebula.js/sn-bar-chart';
+import line from '@nebula.js/sn-line-chart';
+import connect from './connect';
+
+function init() {
+  connect().then(async (app) => {
```

Comment:
as you are not using async await syntax (maybe you should) you should not need any async here